### PR TITLE
[WIP] converting the shared/menu component to es modules

### PR DESCRIPTION
**Pull request by clarkbw**
Associated Issue: #1884

### Summary of Changes

* converted first require to import

### CODE REVIEW

**Comment by gregtatum** [comment]
Hey Bryan, thanks for the PR. The first thing that caught my eye was that you can export individual functions instead of using one big export default. See https://github.com/devtools-html/debugger.html/blob/3e86312a46a0e37450785b27a90224be647a58da/src/utils/editor/expression.js#L7

File: src/components/shared/menu.js [hunk 1]
```diff
@@ -92,7 +92,9 @@ function buildMenu(items: Array<any>) {
     .filter(itm => itm !== null);
 }
 
-export default {
+// export default Dropdown;
```
Comment (clarkbw) [inline src/components/shared/menu.js]: I was trying this but then got console errors.

**Comment by gregtatum** [comment]
https://github.com/devtools-html/debugger.html/blob/3e86312a46a0e37450785b27a90224be647a58da/src/components/App.js#L22-L26

Also, default exports from ES modules require this `.default` at the end of the requires. Check out the example above.

**Comment by jasonLaster** [comment]
Thanks @gregtatum

**Pull request merged**